setup.cfg

```diff
@@ -1,42 +1,19 @@
 [tool:pytest]
 norecursedirs =
-    build
-    cpython
-    emsdk
-    .artifacts
+	build
+	cpython
+	emsdk
+	.artifacts
 addopts =
-    --doctest-modules
-    --ignore="packages/matplotlib/src"
+	--doctest-modules
+	--ignore="packages/matplotlib/src"
```

Comment:
Maybe better to revert this change? In most files we use spaces not tabs.

Reply:
Yeah, this file was updated by `bump2version`.